unlock-app/src/hooks/usePaywallLocks.ts

```diff
@@ -31,6 +32,14 @@ export const usePaywallLocks = (
 
       // getLock doesn't always return the lock address apparently
       lock.address = lockAddresses[index]
+
+      // Use the config's overwrite for the name if applicable
+      Object.keys(config.locks).forEach(configLockAddress => {
+        // The address may differ from the config to the lock received from web3Service.
+        if (configLockAddress.toLowerCase() === lock.address.toLowerCase()) {
```

Comment:
I think this solves the normalization issue.

Reply:
weird. It looks like it does not :/

Reply:
We have a `foreach` over the locks inside a `foreach` over the locks here. I think we can simplify this by just using the lock address already available inside each iteration of the existing loop to index into the config and pull the override name.

Reply:
Good point!